src/sass/shared/mixins.scss

```diff
@@ -132,3 +144,12 @@
         box-shadow: $shadow-color-pill-button-white-hover;
     }
 }
+
+@mixin valign-flex-center($horizontalAlign: false) {
```

Comment:
why does a mixin named `valign-flex-center` also center horizontally?

Also, `@include valign-flex-center` is 27 chars long and `display: flex; align-items: center;` is 34 chars long. I don't know if this is worth a mixin. It adds an indirection where it is not really needed.